doc/explanation/lxd_lxc.md

```diff
@@ -0,0 +1,51 @@
+---
+discourse: 24
+---
+
+# About `lxd` and `lxc`
+
+LXD is frequently confused with LXC, and the fact that LXD provides both a `lxd` command and a `lxc` command doesn't make things easier.
+
+## LXD vs. LXC
+
+LXD and LXC are two distinct implementations of Linux containers.
+
+[LXC](https://linuxcontainers.org/lxc/introduction/) is a low-level user space interface for the Linux kernel containment features.
+It consists of tools (`lxc-*` commands), templates, and library and language bindings.
+
+[LXD](https://linuxcontainers.org/lxd/introduction/) is a more intuitive and user-friendly tool aimed at making it easy to work with Linux containers.
+It is an alternative to LXC's tools and distribution template system, with the added features that come from being controllable over the network.
+Under the hood, LXD uses LXC to create and manage the containers.
+
+LXD provides a superset of the features that LXC supports, and it is easier to use.
+Therefore, if you are unsure which of the tools to use, you should go for LXD.
+LXC should be seen as an alternative for experienced users that want to run Linux containers on distributions that don't support LXD.
+
+(lxd-daemon)=
+## LXD daemon
+
+The central part of LXD is its daemon.
+It runs persistently in the background, manages the instances, and handles all requests.
+The daemon provides a REST API that you can access directly or through a client (for example, the default command-line client that comes with LXD).
+
```

Comment:
Is there anything more we can say about what the daemon does and what role it has?

Reply:
Not really, I mean, it's LXD itself, everything happens in the daemon.